src/cdk/a11y/aria-describer/aria-describer.ts

```diff
@@ -77,8 +98,8 @@ export class AriaDescriber implements OnDestroy {
     if (typeof message !== 'string') {
       // We need to ensure that the element has an ID.
       setMessageId(message);
```

Comment:
The same ID would be assigned twice if the AriaDescriber symbol is packaged twice :
![image](https://user-images.githubusercontent.com/588624/131876603-13fcd806-6ead-464b-a4ad-fd50682e41bd.png)

Reply:
Hmm, how would that happen? We're keeping the counter for unique IDs outside of the class so it'll be global for all instances: https://github.com/angular/components/blob/master/src/cdk/a11y/aria-describer/aria-describer.ts#L36. The only way I can see this happening is if the script was somehow loaded twice.

Reply:
Yes having the library loaded twice may happend (even if not recommended for performance).


*My use case is the following*: I'm developing a prebuilt UI component that can handle payment with various payment methods in a secure way. It's a full angular application wrapped inside a single Javascript file, and that can be used in any website, no matter the framework it uses.
If the website is also using Angular CDK, the AriaDescriber script will be loaded twice.

( Having the CDK_DESCRIBEDBY_ID_PREFIX variable customizable would be enough to work around this issue )

Reply:
Supporting this won't be practical on our end, because even if we did handle it in the `AriaDescriber`, we have many more places that are set up in a similar way and which could break by generating duplicate IDs.